zellij-server/Cargo.toml

```diff
@@ -28,6 +28,7 @@ typetag = "0.1.7"
 chrono = "0.4.19"
 close_fds = "0.3.2"
 sysinfo = "0.22.5"
+tempfile = "3.2.0"
```

Comment:
We use the `tempfile` dependency in the zellij-utils crate, so let's take it from there.
Here's an example of how we do it with another library: https://github.com/zellij-org/zellij/blob/main/zellij-utils/src/lib.rs#L14